src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Security/PermissionsController.cs

```diff
@@ -61,24 +61,30 @@ public Task<ActionResult<bool>> CanUpdateRelease(Guid releaseId)
             return CheckPolicyAgainstEntity<Release>(releaseId, _userService.CheckCanUpdateRelease);
         }
 
-        public class ReleaseStatusPermissionsViewModel
-        {
-            public bool CanMarkDraft = false;
-            public bool CanMarkHigherLevelReview = false;
-            public bool CanMarkApproved = false;
-        }
+        public record ReleaseStatusPermissionsViewModel(
+            bool CanMarkDraft, bool CanMarkHigherLevelReview, bool CanMarkApproved);
 
         [HttpGet("permissions/release/{releaseId:guid}/status")]
         public async Task<ActionResult<ReleaseStatusPermissionsViewModel>> GetReleaseStatusPermissions(Guid releaseId)
         {
             return await _persistenceHelper.CheckEntityExists<Release, Guid>(releaseId)
-                .OnSuccess(release => new ReleaseStatusPermissionsViewModel
+                .OnSuccess(async release =>
```

Comment:
Much nicer!

Reply:
If I see a `.Result` instead of an `await`, I remove it, inline with generally guidance: https://github.com/davidfowl/AspNetCoreDiagnosticScenarios/blob/master/AsyncGuidance.md#avoid-using-taskresult-and-taskwait

Incidentally, there aren't many `.Result`s in non-test code anymore - I spotted one in `ApplicationUserProfileService.cs`, for example. We should consider removing these.